src/components/form.jsx

```diff
@@ -362,7 +362,7 @@ const Form = () => {
                                 )
                             }
                         >
-                            <SwapVert fontSize="large" color="primary" />
+                            <SwapVert fontSize="large" color="#000" />
```

Comment:
I wouldn't hardcode black here. Perhaps `"secondary"` would be a better choice (primary if my proposed change above is implemented)? I'm not sure it would have the desired effect, but it seems to be defined with the black color as main. Perhaps that would mean a black button with white text on it, though? :-)